src/main/java/cubicchunks/asm/CubicChunksMixinConfig.java

```diff
@@ -144,5 +153,46 @@ public static String getNicelyFormattedName(String name) {
         }
         return out.toString();
     }
-
+    
+    private void writeConfigToJson(File configFile) throws IOException {
+        JsonWriter writer = new JsonWriter(new FileWriter(configFile));
+        writer.setIndent(" ");
+        writer.beginArray();
+        for (BoolOptions configOption : BoolOptions.values()) {
+            writer.beginObject();
+            writer.name("name");
+            writer.value(configOption.name());
+            writer.name("description");
```

Comment:
I did not find any mention on how to add a comments of `\\` or `\**\` type in Json files. Even more, I found an info that such comments are not allowed by Json syntax. Existing specifications of JSON format state that JSON must contain tokens and only tokens.
`writer.name("description");` or any other token ignored by parser is acceptable way of adding comments in JSON files.